frontend/packages/shared/src/utils/featureToggleUtils.test.ts

```diff
@@ -57,14 +57,12 @@ describe('featureToggle url', () => {
     window.history.pushState(
       {},
       'PageUrl',
-      '/?featureFlags=customizeEndEvent,shouldOverrideAppLibCheck&persistFeatureFlag=true',
+      '/?featureFlags=shouldOverrideAppLibCheck&persistFeatureFlag=true',
```

Comment:
Maybe add a different feature flag here? To check that _all_ flags are added to session storage? 🤷‍♀️